### PR TITLE
oddz TVL added

### PR DESCRIPTION
**Pull request by manti**
##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):   https://raw.githubusercontent.com/oddz-finance/brand-assets/master/oddz_icons/img_circle.png

image

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
https://www.coingecko.com/en/coins/oddz

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)

https://coinmarketcap.com/currencies/oddz/

##### Short Description (to be shown on DefiLlama): simple, secure and decentralized
Multi-chain derivatives trading platform

### CODE REVIEW

File: projects/oddz/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            target: pool[1],
+            owner: pool[0],
+            chain,
+            block
+        }).then(result => sdk.util.sumSingleBalance(balances, pool[2], result.output))
+    ))
+    return balances
+}
+const bscPools = [
+    // pool, token, representation
```
Comment (Nemusonaneko) [inline projects/oddz/index.js:15]: 0x636f9d2Bb973D2E54d2577b9976DedFDc21E6672 sODDZ pool is considered a staking pool 
0x3c2c77353E2F6AC1578807b6b2336Bf3a3CbB014 ODDZ-BNB is a pool2 LP

Only the 0x99f29c537c70897f60c9774d3f13bd081D423467 oUSD LP would be considered in tvl

**Review by Nemusonaneko — APPROVED** [review]
Will merge and separate pool2 and staking from tvl

**Comment by manti** [comment]
@Nemusonaneko The color scheme is obscured. Could you please resolve this?

Screenshot 2021-11-29 at 11 36 34 AM

**Comment by 0xngmi** [comment]
color is based on the color of your icon but yeah it's not great there

**Comment by manti** [comment]
@0xngmi @Nemusonaneko is it possible to fix this without changing the logo color?

**Comment by manti** [comment]
@0xngmi @Nemusonaneko Any update?

**Pull request merged**